src/Service/AuthService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Domain.Requests;
+using Domain.Responses;
+using Infra.Repositories;
+using Service;
+
+namespace Application.Services

```

Comment:
Se for dar merge na PR anterior, atualizar da main após.
Senão adequar o namespace para Service.